### PR TITLE
Added tasks four chapter

### PR DESCRIPTION
**Pull request by Shillerm**

### CODE REVIEW

File: cycle_for/task_02.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Даны два целых числа A и В. Выведите все числа от A до B включительно, в порядке возрастания, если A < B,
+# или в порядке убывания в противном случае.
+
+A = int(input())
+B = int(input())
+
+if A <= B:
```
Comment (Ken4scholars) [inline cycle_for/task_02.py]: You can do it this way to make it shorter:
```
step = 1 if A < B else -1
for number in range(A, B + step, step):
    print(number)
```

File: cycle_for/task_03.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Даны два целых числа A и В, A>B. Выведите все нечётные числа от A до B включительно, в порядке убывания. В этой
+# задаче можно обойтись без инструкции if.
+
+A = int(input())
+B = int(input())
+
+if A % 2 != 0:
```
Comment (Ken4scholars) [inline cycle_for/task_03.py]: The instruction says you can solve it without `if`. Try that

File: cycle_for/task_04.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Дано 10 целых чисел. Вычислите их сумму. Напишите программу, использующую наименьшее число переменных.
```
Comment (Ken4scholars) [inline cycle_for/task_04.py]: As the task says - you should use the minimal number of variables. You can still reduce it further like this:
```
sum = 0
for i in range(10):
    sum += int(input())
print(sum)
```

File: cycle_for/task_05.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Дано несколько чисел. Вычислите их сумму. Сначала вводите количество чисел N, затем вводится ровно N целых чисел.
+# Какое наименьшее число переменных нужно для решения этой задачи?
```
Comment (Ken4scholars) [inline cycle_for/task_05.py]: Same as last question. You're using way too many variables here. At every iteration, you're creating a new variable `number`

File: cycle_for/task_06.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# По данному натуральному n вычислите сумму 13+23+33+...+n3.
+
+n = int(input())
+
+sum = 0
+for i in range(1, n + 1):
```
Comment (Ken4scholars) [inline cycle_for/task_06.py]: I don't think the solution is correct. Does it pass all the tests?

**Review by Ken4scholars — CHANGES_REQUESTED** [review]
These are called loops in English and not circles. In Russian, they called цикл. I also made that mistake when I started. Good so far, just a little more attention to the task requirements is needed. 
Try to answer to my comments and refactor the code where needed. Don't merge it until I approve.

File: cycle_for/task_02.py (hunk 1 above)
Comment (Shillerm) [reply cycle_for/task_02.py]: Okey

File: cycle_for/task_03.py (hunk 2 above)
Comment (Shillerm) [reply cycle_for/task_03.py]: a = int(input())
b = int(input())
for i in range(a - (a + 1) % 2, b - b % 2, -2):
    print(i, end=' ')

Comment (Shillerm) [reply cycle_for/task_03.py]:
```
a = int(input())
b = int(input())
for i in range(a - (a + 1) % 2, b - b % 2, -2):
    print(i, end=' ')
```

File: cycle_for/task_06.py (hunk 5 above)
Comment (Shillerm) [reply cycle_for/task_06.py]: What's wrong?

File: cycle_for/task_05.py (hunk 4 above)
Comment (Shillerm) [reply cycle_for/task_05.py]:
```
N = int(input())
sum = 0
for number in range(N):
    sum += int(input())
print(sum)
```

File: cycle_for/task_06.py (hunk 5 above)
Comment (Ken4scholars) [reply cycle_for/task_06.py]: @Shillerm I see now. From the comments, I thought you were supposed to calculate 13 + 23 + ... n3 but it's actually the cube - 1^3 + 2 ^3 +...n^3. I checked the question in the site so now I know it is correct

**Review by Ken4scholars — APPROVED** [review]

**Pull request merged**